storm-core/src/clj/backtype/storm/config.clj

```diff
@@ -258,8 +264,8 @@
      (if workers-artifacts-dir
        (if (is-absolute-path? workers-artifacts-dir)
          workers-artifacts-dir
-         (str backtype.storm.util/LOG-DIR file-path-separator workers-artifacts-dir))
-       (str backtype.storm.util/LOG-DIR file-path-separator "workers-artifacts"))))
+         (str backtype.storm.config/LOG-DIR file-path-separator workers-artifacts-dir))
+       (str backtype.storm.config/LOG-DIR file-path-separator "workers-artifacts"))))
```

Comment:
We can just remove the `backtype.storm.config/` from these. They don't need an explicit namespace because they're in the same namespace.